ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -1409,9 +1409,9 @@ class SignalProducerSpec: QuickSpec {
 			}
 
 			describe("disposal") {
-				var completeOuter: (Void -> Void)!
-				var disposeOuter: (Void -> Void)!
-				var execute: (FlattenStrategy -> Void)!
+				var completeOuter: (() -> ())!
+				var disposeOuter: (() -> ())!
+				var execute: (FlattenStrategy -> ())!
```

Comment:
I have to be honest, I hate `()` instead of `Void`, it makes it much more difficult to read. 👎